src/cbc_sdk/platform/alerts.py

```diff
@@ -252,6 +252,29 @@ def update_threat(self, remediation=None, comment=None):
         """
         return self._update_threat_workflow_status("OPEN", remediation, comment)
 
+    @staticmethod
+    def search_suggestions(cb, query):
+        """
+        Returns suggestions for keys and field values that can be used in a search.
+
+        Args:
+            cb (CBCloudAPI): A reference to the CBCloudAPI object.
+            query (str): A search query to use.
+
+        Returns:
+            list: A list of search suggestions expressed as dict objects.
+
+        Raises:
+            ApiError: if cb is not instance of CBCloudAPI
+        """
+        from cbc_sdk.rest_api import CBCloudAPI
```

Comment:
If the import is at the beginning it raises an error for circular import.

Reply:
The other alternative would be using `obj.__class__.__name__`

Reply:
Thanks, I will change that, but probably tomorrow.